ui-core/package.json

```diff
@@ -13,6 +13,7 @@
   },
   "main": "./build/index.js",
   "scripts": {
-    "build": "NODE_ENV=production babel src --copy-files --delete-dir-on-start --extensions .js,.jsx,.ts,.tsx --out-dir build && tsc --declaration --emitDeclarationOnly"
+    "build": "NODE_ENV=production babel src --copy-files --extensions .js,.jsx,.ts,.tsx --out-dir build && tsc --declaration --emitDeclarationOnly",
```

Comment:
`--delete-dir-on-start` makes the admin/participant dev servers fail with "Cannot find module '@juniper/ui-core' or its corresponding type declarations.